tests/Unit/Cli/CampaignUtilizationValidatorTest.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\Cli;
+
+/**
+ * @covers \WMDE\Fundraising\Frontend\BucketTesting\Validation\CampaignUtilizationValidator
+ */
+class CampaignUtilizationValidatorTest extends \PHPUnit\Framework\TestCase {
+
```

Comment:
The test should check if the validator still passes when the buckets and checks are not in the same order (both on campaign and bucket level):

```
campaigns.c1.foo
campaigns.c1.bar
campaigns.c2.baz
campaigns.c2.quux
```

should be similar to

```
campaigns.c2.baz
campaigns.c2.quux
campaigns.c1.foo
campaigns.c1.bar
```

and 

```
campaigns.c1.foo
campaigns.c1.bar
campaigns.c2.baz
campaigns.c2.quux
```

should be similar to

```
campaigns.c1.bar
campaigns.c1.foo
campaigns.c2.quux
campaigns.c2.baz
```

Reply:
I think I have incorporated that now in `newTestCampaignCollection()` but please let me know if there tests can be improved in that regard.

Reply:
👍   The instantiation of buckets in `newTestCampaignCollection` is in a different order than the checks in `ChoiceFactoryMock`, that's good enough.